eland/arithmetics.py

```diff
@@ -33,6 +33,7 @@ class ArithmeticObject(ABC):
     def value(self) -> str:
         pass
 
+    @property
```

Comment:
The sub class implementations have the `@property` decorator, seems like an oversight not to have it here

Reply:
Ah yeah, likely an oversight. Thanks!